### PR TITLE
Fixed issues as per PR review.

### LINKED ISSUE #2: Create note listing page UI.

### LINKED ISSUE #10: Fix issues according to PR review.

### PR DESCRIPTION
**Pull request by sunnyshahabuddin**
@yedhink Please review.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 1]
```diff
@@ -44,7 +44,6 @@ const Note = ({ note }) => (
           user={{
             imageUrl: "https://xsgames.co/randomusers/avatar.php?g=pixel",
           }}
-          onClick={function noRefCheck() {}}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:47]: Check the browser console and see if there are any warnings. I believe these components should be having an onClick mandatorily. But if that's not the case, then no issues.

Refer: https://academy.bigbinary.com/learn-react/pages/prefer-es6-arrow-functions#:~:text=Always%20prefer%20arrow%20functions

**Review by yedhink — COMMENTED** [review]
@sunnyshahabuddin

File: app/javascript/src/components/Dashboard/Notes/Note.jsx (hunk 1 above)
Comment (sunnyshahabuddin) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx:47]: There is a warning in my console, which even after adding `onClick={() => {}}` in the Avatar component persists. I am attaching the screenshot of the warning. However there is no such warnings related to the `onClick` function.
Screenshot 2022-09-20 at 7 49 55 PM

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx:47]: You can ignore that warning.

Comment (sunnyshahabuddin) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx:47]: Thank you Yedhin.

**Pull request merged**